docs/pages/packaging.md

```diff
@@ -3,7 +3,22 @@ title: Packaging
 layout: default
 ---
 
-# Packaging
+# General packaging
+
+A common question facing developers is "_How much code should go into a
+package?_". Where code to solve a research problem might be large and perform
+several tasks at once typically one should try to stick to a mantra of doing one
+thing well.
```

Comment:
Probably having some idea of scope before you start writing your package would be my main piece of advice from BrainGlobe, though you've kind of hinted at that in the new text already. Could maybe add

```suggestion
several tasks at once typically one should try to stick to a mantra of doing one
thing well.
The following questions can be helpful when trying to abide by this mantra:
- What do I want my package to provide?
- How will users interact with or use the code in my package?
- Is everything I'm including in the package relevant or useful in supporting it's main purpose? 
```